nginx_proxy/start_nginx.sh

```diff
@@ -79,12 +97,16 @@ if [[ -f "${CONF_FILE}" ]]; then
   cp "${CONF_FILE}" /etc/nginx/nginx.conf
 fi
 
-# fetch Service Configuration from Service Management if the service name and
-# service version are provided.
-if [[ -n "${ENDPOINTS_SERVICE_NAME}" && \
-      -n "${ENDPOINTS_SERVICE_VERSION}" ]]; then
-  /usr/sbin/fetch_service_config.sh \
-    -s "${ENDPOINTS_SERVICE_NAME}" -v "${ENDPOINTS_SERVICE_VERSION}" || exit $?
+# Building nginx startup command
+cmd="/usr/sbin/start_esp"
+cmd+=" -n /etc/nginx/nginx.conf"
+cmd+=" -s \"${ENDPOINTS_SERVICE_NAME}\""
+if [[ "${ENDPOINTS_SERVICE_VERSION}" ]]; then
+  cmd+=" -v \"${ENDPOINTS_SERVICE_VERSION}\""
+fi
+if [[ "${ENDPOINTS_ROLLOUT_STRATEGY}" ]]; then
+  cmd+=" --rollout_strategy \"${ENDPOINTS_ROLLOUT_STRATEGY}\""
```

Comment:
Should this be --rollout_strategy or just -r?

Reply:
start_esp already has -r for another purpose. -R is the short option and --rollout_strategy should work.